client/lib/discounts/active-discounts.js

```diff
@@ -70,11 +68,9 @@ export default [
 		],
 	},
 	{
-		name: 'sale_feb_2019_jp',
-		startsAt: new Date( 2019, 1, 25, 0, 0, 0 ),
-		endsAt: new Date( 2019, 2, 8, 0, 0, 0 ),
-		nudgeText: 'Sale! 20% off all plans',
-		ctaText: 'Upgrade',
+		name: 'sale_wpsave20_jp',
+		startsAt: new Date( 2019, 4, 27, 0, 0, 0 ),
```

Comment:
Just noting that this one won't start until 27th of this month, while the other one started last month. I assume the other one was changed for testing. Doesn't matter much since it'll be evergreen

Reply:
:+1: Fixed in https://github.com/Automattic/wp-calypso/pull/33206/commits/c2191f3ee11f4aea08211c3e5c77658fc950f405